d2l-input-textarea.html

```diff
@@ -206,6 +211,10 @@
 			 * @type {string|number}
 			 */
 			value: {observer: '_valueChanged', type: String, notify: true},
+			/**
+			 * Removes left and right padding from textarea.
+			 */
+			noPadding: {type: Boolean, value: false},
```

Comment:
Should really have `reflectToAttribute: true` as well, so that if the property changes (through JavaScript), the corresponding attribute will be updated accordingly.  If this is not present elsewhere, then that scenario probably wasn't tested.